app-arch/tar/files/tar-1.34-fix-cve-2022-48303.patch

```diff
@@ -0,0 +1,30 @@
+From 3da78400eafcccb97e2f2fd4b227ea40d794ede8 Mon Sep 17 00:00:00 2001
```

Comment:
Please include at the top of the patch:
- A link to the relevant Gentoo bug
- A link to the upstream commit

Reply:
Done.